CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Simulation/ViralInfectionVTM.py

```diff
@@ -59,4 +59,8 @@
 
 CompuCellSetup.register_steppable(steppable=oxidationAgentModelSteppable(frequency=1))
 
+from Models.DrugDosingModel.DrugDosingModelSteppable import DrugDosingModelSteppable
+
+CompuCellSetup.register_steppable(steppable=DrugDosingModelSteppable(frequency=1))
+
```

Comment:
We don't want to always do drug therapy simulations in the main framework, so we can't have this here. However, it would be nice to distribute an example simulation within this module, in a subdirectory of `Models/DrugDosingModel`.